pkg/ebpf/common/http/elasticsearch.go

```diff
@@ -67,42 +74,29 @@ func parseElasticsearchRequest(req *http.Request) (elasticsearchOperation, error
 	if err != nil {
 		return op, fmt.Errorf("failed to read Elasticsearch request body %w", err)
 	}
-
 	req.Body = io.NopCloser(bytes.NewBuffer(reqB))
-	if len(reqB) == 0 {
-		op.DBQueryText = ""
-	} else {
-		dbQueryText, err := extractDBQueryText(reqB)
-		if err != nil {
-			return op, err
-		}
-		op.DBQueryText = dbQueryText
-	}
-	op.DBOperationName = extractOperationName(req)
+	operationName := extractOperationName(req)
+	op.DBQueryText = string(reqB)
+	op.DBOperationName = operationName
 	op.DBCollectionName = extractDBCollectionName(req)
 	return op, nil
 }
 
-func extractDBQueryText(body []byte) (string, error) {
-	var buf bytes.Buffer
-
-	if err := json.Compact(&buf, body); err != nil {
-		return "", fmt.Errorf("invalid Elasticsearch JSON body: %w", err)
-	}
+func isSupportedRequest(req *http.Request) error {
+	urlPath := req.URL.Path
 
-	return buf.String(), nil
-}
+	for operation, methods := range supportedOperationMethods {
+		if strings.Contains(urlPath, operation) {
```

Comment:
is there a way to match on the last part of the URL instead of using `strings.Contains`?

Reply:
perhaps we can check the last and second last part? not sure how ugly it is or if it covers the cases we need